tools/dev/repo-health.go

```diff
@@ -65,7 +65,6 @@ func replaceInFile(fileName string, marker string, content string) error {
 	}
 	endMarkerIdx += startMarkerIdx + 1
 	out.WriteString(strings.Join(lines[endMarkerIdx:], "\n"))
-	out.WriteString("\n")
```

Comment:
Isn't this enough?

Reply:
Yes the quotes are a minor improvement on zsh